### PR TITLE
Reword First Block page

### PR DESCRIPTION
**Pull request by Rouxles**

### CODE REVIEW

File: src/fb.md [hunk 1]
```diff
@@ -2,51 +2,53 @@
...
 
-- [Movecount statistics](https://docs.google.com/spreadsheets/d/1EectP3O_qwQp_2WohbDrlohxhs2ee17sjFfxZr_-D1g/edit#gid=0) - Notice how x2y boosts chance of easy First Sqaure compared to y from 60% to 80%.
+If you are interested in the average movecount comparison of FB or FB + DR under different orientations, use the following resource:
+
+- [Movecount statistics](https://docs.google.com/spreadsheets/d/1EectP3O_qwQp_2WohbDrlohxhs2ee17sjFfxZr_-D1g/edit#gid=0) - Notice how x2y boosts chance of easy first square compared to y from 60% to 80%.
 
 ## Blockbuilding Strategies
 
-There are broadly speaking two families of FB building strategy: Square + pair, and line + line. See resource below:
+There are broadly speaking two families of FB building strategy: square + pair and line + line. See resources below:
```
Comment (Rouxles) [inline src/fb.md:20]: why is this lowercase but then the things in line 24/27/29 full caps?

Comment (Rouxles) [reply src/fb.md:20]: is this intended or nah

File: src/fb.md [hunk 2]
```diff
@@ -2,51 +2,53 @@
...
 
-- Also, develop a preference for what pairs / triggers are easy to execute over time, and try to prioritize them in your FB solution. Use this to guide your FB planning instead of move count as this is more precise and correlated to actual times.
+- Also, develop a preference for what pairs / triggers are easy for you to execute over time, and try to prioritize them in your FB solution. Use this to guide your FB planning instead of move count as this is more precise and correlated to actual times.
 
 ## How To Bring your inspection down to 15 seconds
 
 
-- [Partial SpeedBLD technique by Kian](https://www.youtube.com/watch?v=4KLFyN6ZDwk) - This is definitely underrated as a practice approach! Do them regularly, as warmups before solves!
+- [Partial SpeedBLD technique by Kian](https://www.youtube.com/watch?v=4KLFyN6ZDwk) - This is definitely underrated as a practice approach! Do them regularly, as warmup before solves!
```
Comment (Rouxles) [inline src/fb.md:54]: maybe remove the comma here?

**Comment by Rouxles** [comment]
other than that everything lgtm

**Review by V-Wong — APPROVED** [review]

**Pull request merged**